api/HTMLBaseElement.json

```diff
@@ -81,6 +81,39 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "forbid_data_javascript_urls": {
+          "__compat": {
+            "description": "<code>data:</code> and <code>javascript:</code> urls are not allowed",
+            "support": {
+              "chrome": {
+                "version_added": false
```

Comment:
Can you add https://issues.chromium.org/issues/40928991 otherwise we will probably never find out if/when this gets fixed in Chrome.

Reply:
```suggestion
                "version_added": false,
                "notes": "See <a href='https://issues.chromium.org/issues/40928991>issue 40928991</a>."
```

Reply:
Thanks!